tests/testfont.c

```diff
@@ -0,0 +1,55 @@
+#ifdef WIN32
```

Comment:
I think it makes more sense to keep the file singular and just update the makefile to be singular too